gnosis/safe-react
# [Address Book V2] Import address book

**What**:

Closes #2236

**Why**:

We are adding the import of CSV files to let the user load entries to the address book

**How**:

Added a modal that let's you load a single file and gives you feedback on how many entries have been loaded

**How to test it**:

- Enter the address book page
- Click on the top right Import button
- Load a CSV file (also supports xls and xlsx files) that has two columns (address, name)
- Click on import
- Entries should be added and a notification should appear

**Checklist**:

- [ ] Ready to be merged "N/A"

Right now the entries are saved using `addAddressBookEntry()`. This will not overwrite the name of an entry that's already in the store. This will be changed on the store refactor.
